tox.ini

```diff
@@ -114,6 +115,28 @@ commands =
     rm -rf {envdir}/.pytest_cache
     make regen
 
+[testenv:plugins]
+changedir = testing/plugins_integration
+deps =
+    django
+    pytest-asyncio
+    pytest-cov
+    pytest-django
+    pytest-html
+    pytest-mock
+    pytest-sugar
+    pytest-trio
```

Comment:
thoughts on a `pytest-anyio` test?

Reply:
Sounds good, feel free to push a new test to this branch. 👍